crates/lib/test/fixtures/dialects/bigquery/dateparts.yml

```diff
@@ -245,7 +245,7 @@ file:
             - function_name_identifier: EXTRACT
           - bracketed:
             - start_bracket: (
-            - date_part: DATE
+            - warehouse_size: DATE
```

Comment:
`warehouse_size` here feels odd, but maybe I have misunderstood the Select context.